psychopy/app/runner/scriptProcess.py

```diff
@@ -232,6 +254,32 @@ def _onTerminateCallback(self, pid, exitCode):
                 self.expCtrl.Select(itemIdx)
                 self.runBtn.Enable()
 
+        def _focusOnOutput(win):
+            """Subroutine to focus on a given output window."""
+            win.Show()
+            win.Raise()
+            win.Iconize(False)
+
+        # set focus to output window
+        if self.app is not None:
+            if self.focusOnExit == 'coder' and hasattr(self.app, 'coder'):
```

Comment:
I'd quite like to take this further at some point - and have `.builder`, `.coder` and `.runner` as a `@property` which returns the window if open, or opens the window if not.